### PR TITLE
New JIRA Cheatsheet

### PR DESCRIPTION
**Pull request by jophab**
**What does your Instant Answer do?**
Shows the quick reference of jira s/w

**What problem does your Instant Answer solve (Why is it better than organic links)?**
Shows the quick reference of jira s/w

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://www.cheatography.com/ovi-mihai/cheat-sheets/jira/

**Why did you choose this data source?**
Its popular

**Are there any other alternative (better) data sources?**
may be.

**What are some example queries that trigger this Instant Answer?**
cpp help, c++ help

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
s/w  developers

**Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?**
no

**Which existing Instant Answers will this one supercede/overlap with?**
none

**Are you having any problems? Do you need our help with anything?**
no

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

-----
IA Page: https://duck.co/ia/view/jira_cheat_sheet

### CODE REVIEW

**Comment by jophab** [comment]
@moollaza @codenirvana Why it failed the checks?

File: share/goodie/cheat_sheets/json/jira.json [hunk 1]
```diff
@@ -0,0 +1,166 @@
+{
+    "id": "jira_cheat_sheet",
+    "name": "JIRA",
+    "description": "JIRA shortcuts",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/jira.json]: Let's make this a little more descriptive, or simply remove it, if it doesn't add much value

**Comment by moollaza** [comment]
> Why it failed the checks?

@jophab it's an internal issue we're working to fix. The Cheat Sheet should be fine. You can run `duckpan test` locally to make sure everything is OK

**Comment by gautamkrishnar** [comment]
@jophab deployed to beta :+1:

**Comment by daxtheduck** [comment]
Automated screenshot from https://beta.duckduckgo.com

File: share/goodie/cheat_sheets/json/jira.json [hunk 2]
```diff
@@ -0,0 +1,165 @@
+{
+    "id": "jira_cheat_sheet",
+    "name": "JIRA",
+    "metadata": {
+        "sourceName": "cheatography",
+        "sourceUrl": "http://www.cheatography.com/ovi-mihai/cheat-sheets/jira/"
+    },
+    "template_type": "reference",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/jira.json]: This should be using the `keyboard` template seeing as it's all keyboard shortcuts :+1:

File: share/goodie/cheat_sheets/json/jira.json [hunk 3]
```diff
@@ -0,0 +1,165 @@
+{
+    "id": "jira_cheat_sheet",
+    "name": "JIRA",
+    "metadata": {
+        "sourceName": "cheatography",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/jira.json]: This should be cased properly `Cheatography`

File: share/goodie/cheat_sheets/json/jira.json (hunk 2 above)
Comment (moollaza) [inline share/goodie/cheat_sheets/json/jira.json]: You'll need to flip the `keys` and `vals`

**Comment by daxtheduck** [comment]
Automated screenshot from https://beta.duckduckgo.com

**Comment by daxtheduck** [comment]
Automated screenshot from https://beta.duckduckgo.com

**Comment by jophab** [comment]
@moollaza  @mintsoft  automated screen shot is not working correctly.
It still shows the template 'reference' eventhough I changed it to 'keyboard'.

**Comment by moollaza** [comment]
@jophab this LGTM :+1:

**Comment by moollaza** [comment]
@jophab congrats! It's live: https://duckduckgo.com/?q=JIRA+Cheat+Sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**